### PR TITLE
test against ruby 2.4.2

### PR DESCRIPTION
**Pull request by siegy22**
Hey @stadelmanma 

I want to use sablon with ruby 2.4.2, so let's also test against that now.

### CODE REVIEW

**Comment by stadelmanma** [comment]
@siegy22 good call on adding 2.4 to the test suite, I've been using it for awhile now but forgot to add it to Travis CI. I don't think we need to pin the ruby version down to be patch specific judging by the example in the Travis docs and if not I'd rather relax that constraint since I can't think of a good reason to pin things back unnecessarily.

Since were already modifying the CI config would you mind to go ahead and update the rvm section to look like this?
```yaml
rvm:
  - 2.1
  - 2.2
  - 2.3
  - 2.4
```

After that I'll go ahead and merge this.

File: .travis.yml [hunk 1]
```diff
@@ -3,3 +3,4 @@ rvm:
   - 2.1.8
```
Comment (stadelmanma) [inline .travis.yml]: only pin all ruby version by major and minor, no need to be patch specific

**Comment by siegy22** [comment]
@stadelmanma done :+1:

**Comment by stadelmanma** [comment]
@siegy22 Thanks!  :+1:

**Pull request merged**